src/test/java/org/apache/sling/distribution/agent/impl/PrivilegeDistributionRequestAuthorizationStrategyTest.java

```diff
@@ -81,17 +95,19 @@ public void testNoPermissionOnAdd() throws Exception {
     }
 
     @Test
-    public void testPermissionOnAdd() throws Exception {
+    public void testDefaultPermissionOnAdd() throws Exception {
         String jcrPrivilege = "somePermission";
-        PrivilegeDistributionRequestAuthorizationStrategy strategy = new PrivilegeDistributionRequestAuthorizationStrategy(jcrPrivilege);
+        String DEFAULT_READ_PRIVELEGE = "jcr:read";
```

Comment:
Do we really need this? Can't we just use Privilege.JCR_READ?
If we can't, please fix the typo (misspelt "privilege")